src/test/javascript/general.controller.test.js

```diff
@@ -381,6 +381,15 @@ describe("GeneralController", function () {
         });
     });
 
+    // describe("closeApiError", () => {
```

Comment:
Is there a good reason for commenting this out? If so, please document the reason in the comment.

Reply:
Same thing for the other sections which are commented out.

Reply:
Gotcha, idk why this is even here